ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/audit/audit-endpoint.tsx

```diff
@@ -0,0 +1,33 @@
+import fetch from '../fetch'
+
+export type AuditLog = {
+  action: string
+  component: string
+  ids: Set<string> | string
+}
+
+export const postAuditLog = async ({ action, component, ids }: AuditLog) => {
+  if (typeof ids === 'string') {
```

Comment:
Does this case still happen?

Reply:
Not at the moment, but I added this logic here so that anyone could call this function and pass in a set of strings or just a string. I didn't want anyone to have to remember the parameter details.